src/AutomatedCar/Views/MainWindow.xaml.cs

```diff
@@ -1,15 +1,30 @@
 namespace AutomatedCar.Views
 {
+    using System;
+    using System.Reactive.Linq;
     using AutomatedCar.ViewModels;
     using Avalonia.Controls;
     using Avalonia.Input;
     using Avalonia.Markup.Xaml;
+    using Avalonia.ReactiveUI;
+    using ReactiveUI;
 
-    public class MainWindow : Window
+    public class MainWindow : ReactiveWindow<MainWindowViewModel>
     {
         public MainWindow()
         {
             this.InitializeComponent();
+            var popUpWindow = new PopUpWindow();
+            popUpWindow.Closing += (s, e) =>
+            {
+                ((Window)s).Hide();
+                e.Cancel = true;
+            };
```

Comment:
Ezt az egy sort még a popUpWindow.Closing-hoz hozzáadod légyszi?
A popup valamiért bezavar a MainWindowba és beragadnak az ütközés előtt lenyomott billenytűk
```suggestion
            popUpWindow.Closing += (s, e) =>
            {
                ((Window)s).Hide();
                e.Cancel = true;
                Keyboard.Keys.Clear();
            };
```

Reply:
Beraktam, köszi az észrevételt!